js/combobox.js

```diff
@@ -9,6 +9,16 @@ $(function() {
          this._createAutocomplete();
          this._createShowAllButton();
          this.input.attr("placeholder", this.element.attr('placeholder'));
+
+         this.input.keydown(function(event) {
+           var enterkey = 13;
+           if(event.keyCode === enterkey) {
+             var $firstMenuItem = $($('li')[0]);
```

Comment:
I would try:

```js
$('li:first:visible').click()
```

if that doesn't work, I'd try:

```js
$('li').filter(':first').filter(':visible').click()
```

see [jquery :first selector](https://api.jquery.com/first-selector/) and [jquery :visible selector](https://api.jquery.com/visible-selector/)

Also it's important to understand that by calling `$('li')` you will select *every* `li` on the page, which might not be what you want... I find it helpful to specify a parent element in the selector so that it applies only to the specific `li`s that you want to target.